python/mxnet/kvstore.py

```diff
@@ -166,6 +166,9 @@ def push(self, key, value, priority=0):
         There is no synchronization between workers.
         One can use ``_barrier()`` to sync all workers.
 
+        Note: This api is not supported for kvstore with type dist_sync_mpi. Use pushpull
```

Comment:
nit: we can improve it with 
```
:py:meth:`pushpull`
```
so that the doc renders with nice hyperlinks. See https://mxnet.incubator.apache.org/versions/master/api/python/gluon/gluon.html#mxnet.gluon.ParameterDict.get_constant for example

Reply:
OK.